src/user/serializers.py

```diff
@@ -61,6 +61,7 @@ class AuthorSerializer(ModelSerializer):
     total_score = SerializerMethodField()
     university = UniversitySerializer(required=False)
     wallet = SerializerMethodField()
+    status = SerializerMethodField()
```

Comment:
IMO we should call this something else, like `suspended_status`

Reply:
yeah that sounds better